diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index 41f5d8242..a47cfee6a 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1813,6 +1813,7 @@ u64 snd_usb_interface_dsd_format_quirks(struct snd_usb_audio *chip,
 	case 0x152a:  /* Thesycon devices */
 	case 0x20b1:  /* XMOS based devices */
 	case 0x22d9:  /* Oppo */
+	case 0x22e8:  /* Cambridge */
 	case 0x23ba:  /* Playback Designs */
 	case 0x25ce:  /* Mytek devices */
 	case 0x278b:  /* Rotel? */
