diff --git a/arch/arm64/boot/dts/amlogic/kvim3l_linux.dts b/arch/arm64/boot/dts/amlogic/kvim3l_linux.dts
index 247bb567..c09cb1c5 100644
--- a/arch/arm64/boot/dts/amlogic/kvim3l_linux.dts
+++ b/arch/arm64/boot/dts/amlogic/kvim3l_linux.dts
@@ -572,7 +572,7 @@
 
 	auge_sound {
 		compatible = "amlogic, g12a-sound-card";
-		aml-audio-card,name = "AML-AUGESOUND";
+		aml-audio-card,name = "AML-AUGESOUND-V";
 
 		aml-audio-card,dai-link@0 {
 			mclk-fs = <128>;
